ironfish/src/network/peerNetwork.ts

```diff
@@ -368,11 +369,25 @@ export class PeerNetwork {
       // it's running on the default ironfish websocket port
       const port = url.port ? url.port : DEFAULT_WEBSOCKET_PORT
 
-      this.peerManager.connectToWebSocketAddress({
+      const bootstrapNodePeer = this.peerManager.connectToWebSocketAddress({
         host: url.hostname,
         port,
         whitelist: true,
       })
+
+      const warnNetworkMismatch = (message: NetworkMessage) => {
+        if (message instanceof IdentifyMessage) {
+          if (message.networkId !== this.networkId) {
+            this.logger.warn(
+              `Bootstrap node ${node} is on network ${message.networkId} while we are on network ${this.networkId}`,
```

Comment:
What do you think of printing the network name for the default networks? Similar to how we do it in the status command? https://github.com/iron-fish/ironfish/blob/master/ironfish-cli/src/commands/status.ts#L127-L128

Reply:
I think that's an excellent idea!